stof/doctrine-extensions-bundle/1.2/etc/packages/stof_doctrine_extensions.yaml

```diff
@@ -0,0 +1,21 @@
+stof_doctrine_extensions:
+    default_locale: en_US
+
+    # Activate the extensions you want
+    orm:
+        default:
+            tree: true
+            timestampable: false # not needed: listeners are not enabled by default
+        other:
+            timestampable: true
+
+#    # Overriding the listeners¶
```

Comment:
I don't know if it's very common to override listeners, is it? If not, then this is unnecessary because the user will definitely need the docs to override a listener, therefore defaulting the listeners classes is useless IMO